runtime/directives/sky-app-link-external.directive.ts

```diff
@@ -0,0 +1,36 @@
+import { Directive, Input, HostListener, OnInit } from '@angular/core';
+import { SkyAppWindowRef } from '../window-ref';
+import { SkyAppConfig } from '@blackbaud/skyux-builder/runtime';
+import { HttpParams } from '@angular/common/http';
+
+@Directive({
+  selector: '[skyAppLinkExternal]'
```

Comment:
Make the selector more specific to only allow anchor tags (i.e. `a[skyAppLinkExternal]`) until there's a compelling use case for non-anchor elements.